lib/zendesk_apps_support/package.rb

```diff
@@ -226,6 +223,16 @@ def iframe_only?
 
     private
 
+
+    def runtime_translations(translations)
+      result = translations.dup
```

Comment:
Ah. My bad. Sorry for the wrong suggestion. Just looked at source code of Hash#except and dup was used there as well.